### ISSUE TITLE
Upgrade bootstrap version and related JS libraries

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#348]
It has been a while since last update of jquery, bootstrap, flask and all third party frontend code. We need to upgrade them.

**Comment by pietrosperoni** [comment]
My understanding was that people do not upgrade external code as this can break all sorts of dependencies. Which is why there are such things as dockers where everything is kind of frozen in time. Or did I got it all wrong?

**Comment by giovanisp** [comment]
So you remain frozen in time with all security holes ? :-p
Our code should not rely on deprecated functions of external libraries, so we can upgrade them on latest security patches and functionality. Part of our work is ensuring we keep updated with latest upgrades whenever it brings us more security or more functionality we need to use. eventually we will need to remove deprecated code. It is part of maintaining the codebase.

**Comment by pietrosperoni** [comment]
I agree with you. Does @polites75 and @tommasosperoni also agree with you?
